logging/README.md

```diff
@@ -261,3 +261,13 @@ deployment process.
 * Select the __Discover__ icon in the toolbar to display the Discover page. Use
 this page to review the collected log messages. You can use the query box or
 __Add filter__ to filter the messages that are displayed.
+
+## Enable Users to Access Elasticsearch (optional)
+
+You can choose to enable users to access Elasticsearch so that they can issue queries using API calls or scripts. Run this command to enable nodeport connections to Elasticsearch:
+
+```bash
+./logging/bin/es_nodeport_enable.sh
+``` 
+
```

Comment:
When the es_nodeport_enable.sh script completes, it writes a message with the URL you can use to access Elasticsearch (similar to the message w/the Kibana URL).  For example:

```
=============================================
== Access Elasticsearch using this URL: https://myK8snode:32168/ ==
=============================================
```

However, unlike in the Kibana message, the port number is not fixed (i.e. not always 32168) and users would need to use the port number shown in the message.  Notice the protocol is https (indicating the TLS is enabled) and that this is always the case.
Probably should mention that there is a corresponding script to disable the nodeport enabled by running this script.  Unimaginatively, the disable script is /logging/bin/es_nodeport_disable.sh.  We might also want to mention that these scripts can be useful if users need to temporarily enable access for some reason.

Reply:
Message and note about disable script added

Reply:
Thanks.  One small nit:  The formatting of the Elasticsearch message and the Kibana message shown elsewhere in the doc are not consistent.  While the port number is variable here (compared to the Kibana case where it is hard-coded) and, may, therefore, warrant different treatment,  the way the hostname is shown differs b/w the two examples too.  I have a slight preference for the way things are shown w/the Kibana message but am flexible and am more concerned about consistency.

Reply:
Changed message format and wording to be more consistent with Kibana example